content/docs/introduction/billing.md

```diff
@@ -0,0 +1,199 @@
+
+### Download invoices
+
+You can download invoices from the **Billing** page.
+
+1. Navigate to the **Billing** page in the Neon Console. The current invoice is displayed.
+1. Download the invoice:
+    1. To download the current invoice, click **Download PDF** from the top of the page.
+    1. To download an invoice for a previous billing period, select the invoice from the **Latest invoices** list to open it, then click **Download PDF**.
+
+## Cancel a subscription
+
+To cancel your subscription to a Neon paid plan:
+
+1. Navigate to the **Billing** page in the Neon Console.
+1. Click **Cancel subscription**.
+1. Enter your cancellation request and click **Submit**.
+
+This action initiates the cancellation. If your data exceeds  free-tier storage limits, you will be contacted by the Neon Support team with a request to reduce your storage before the paid plan is canceled and free-tier limits are applied.
+
+## Billing metrics explained
+
+This section provides a detailed explanation of Neon's billing metrics and how they are calculated. Billing in Neon is account-based. If you require a project-based cost breakdown, refer to your [billing invoice](#neon-invoices).
+
+### Compute time
+
+The _Compute time_ metric counts _Compute Unit (CU)_ active time, in hours. In Neon, a compute endpoint can have up to 8 CUs. A connection from a client or application activates a compute endpoint and its CUs. Activity on the connection keeps the compute endpoint and its CUs in an active state. A defined period of inactivity places the compute endpoint and its CUs into an idle state.
+
+Factors that affect the amount of compute time include:
+
+- The number of active compute endpoints
+- The number of CUs per compute endpoint
+- Neon's _auto-suspend compute_ feature, which suspends a compute endpoint (and its CUs) after a specified period of inactivity (5 minutes).
+- Neon's _autoscaling_ feature, which allows you to set a minimum and maximum number of CUs for each compute endpoint. The number of active CUs scale up and down based on workload. _This feature is not yet available._
+- Neon's _always-on_ compute feature, which keeps one endpoint active indefinitely to avoid connection latency due compute endpoint startup time. _This feature is not yet available._
+
+The cost calculation for _Compute time_ is as follows:
+
+```text
+compute units * active time (hours) * price per hour
+```
+
+### Project storage
+
+The _Project storage_ metric counts the amount of data stored in all of your Neon projects. Project storage includes:
+
+- **The logical data size of your data**
+
+  This includes the size of all databases in your Neon projects, PostgreSQL SLRU (simple least-recently-used) caches, and a small amount of metadata. You can think of this as a _snapshot_ of your data at a point in time and similar to the data size that you would see in a standalone PostgreSQL installation.
```

Comment:
To be precise, currently we do not account SLRU size in logical size.

from pageserver code:
```    /// Does the same as get_current_logical_size but counted on demand.
    /// Used to initialize the logical size tracking on startup.
    ///
    /// Only relation blocks are counted currently. That excludes metadata,
    /// SLRUs, twophase files etc.
    pub async fn get_current_logical_size_non_incremental(
 ```
    
I think this should be fixed in the code, not in the documentation.
@stepashka , should we create an issue for that and prioritize it?

Reply:
@stepashka Should I remove mention of SLRU caches and metadata?